manifests/master/puppetdb_conf.pp

```diff
@@ -3,6 +3,7 @@
 class puppetdb::master::puppetdb_conf (
   $server             = 'localhost',
   $port               = '8081',
+  $servers            = [],
```

Comment:
I think this should probably be called `server_urls` to indicate that it need to be a list of urls rather than just hostnames.